packages/components/app/styles/components/button.scss

```diff
@@ -26,6 +27,15 @@ $hds-button-focus-border-width: 3px;
   // the <a> element behaves differently than a <button>
   @at-root a#{&} {
     width: fit-content;
+
+    &:hover,
```

Comment:
please add a comment to some reference (probably the RFC) of why we're doing this.

Reply:
update now – do let me know if it doesn't read well